examples/flutter_gallery/lib/demo/material/search_demo.dart

```diff
@@ -133,6 +133,12 @@ class _SearchDemoState extends State<SearchDemo> {
   }
 }
 
+/// [SearchDelegate.buildResults] might be called multiple times. Please cache
```

Comment:
There's no need to reproduce the API doc in the demo.

Reply:
Removed.